JMdictFiles/JMdict_to_JSON.py

```diff
@@ -0,0 +1,51 @@
+import xml.etree.ElementTree as ET
+import json
+import re
+
+# Regular expression pattern to match kanji characters
+# [\u4e00-\u9faf] is a range for common kanji characters
+kanji_pattern = re.compile(r'[\u4e00-\u9faf]')
+
+# Parse the XML file
+tree = ET.parse('JMdict_e')
```

Comment:
do we have this file in the repo? There seems to be some inconsistency, lets think what is required for a fellow contributor to work:
1. If we have this `JMdict_to_JSON.py` script then we should have `JMdict_e` file to work with. One might argue that having the `JMdict.json` is now unnecessary because we can generate it on demand from the former files.
2. Alternatively, if we already have `JMdict.json`, why is there a need for `JMdict_to_JSON.py` when we cannot use it without `JMdict_e` ?

Reply:
I was on edge about deleting the files altogether, because I've put considerable time for them to work the way I wanted. If there wasn't a 50MB cap on files in github, `JMdict_e` would have been in the repo too. I see value in sharing these files, because i
I see 3 possible solutions:
1. Add a link to `JMdict_e` download page above line 10
2. Try to make `JMdict_e` weigh < 50MB
3. Untrack all files related to `JMdict_e` - least favourite of mine, because I made dic.js from a part of `JMdict_e`, which has like 130k+ records
Which one should we go with? My personal favourite is #1

Reply:
Decided to remove everything related to JMdict